WordPress/Classes/Networking/AccountServiceRemote.h

```diff
@@ -49,6 +49,18 @@
                  failure:(void (^)(NSError *error))failure;
 
 /**
+ *  @brief      Query to see if a username is available. Used in the auth link signup flow.
+ *  @note       This is an unversioned endpoint. Success will mean, generally, that the username already exists.
+ *
+ *  @param username
+ *  @param taken
+ *  @param available
+ */
+- (void)checkUsernameAvailability:(NSString *)username
+                 taken:(void (^)())taken
+                 available:(void (^)())available;
```

Comment:
Our convention for services is success and failure blocks.  Even though the `is-available` endpoint is special I'm not seeing a strong case for deviating from convention.